liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -593,11 +593,14 @@ public static boolean isNotEmpty(String value) {
     }
 
     /**
-     * Checks if <code>value</code> starts with <code>startsWith</code>.
+     * Checks whether the given <code>value</code> starts with the specified <code>startsWith</code> string.
      *
-     * @param value
-     * @param startsWith
-     * @return true if <code>value</code> starts with <code>startsWith</code>, otherwise false. If any of arguments is null returns false
+     * @param value      the string to check
+     * @param startsWith the prefix to check for
+     * @return <code>true</code> if <code>value</code> starts with <code>startsWith</code>, <code>false</code> otherwise.
+     * css
```

Comment:
css and Copy code seem sort of random. Typos??

And, should the `<code></code>` blocks be `@code` instead?

Reply:
Removed "css" and "copy code". Regarding <code> vs {@code} I replied in my previous comment. My understanding is they both has the same purpose but {@code} does not resolve indentation and line break issues, so maybe we can use any of them depending on each individual case or we can make an agreement and use one of them most of the times.

Reply:
:) Thanks for getting rid of the "css" and "copy code" lines. If both `<code>` and `{@code}` perform the same function, I don't mind which we use.